.github/workflows/NIGHTLY.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         CSC_LINK: "${{ secrets.CSC_LINK }}"
         CSC_KEY_PASSWORD: "${{ secrets.CSC_KEY_PASSWORD }}"
         ET_ENDPOINT: "${{ secrets.ET_ENDPOINT }}"
-        MIXPANEL_TOKEN: "${{ secrets.MIXPANEL_TOKEN }}"
+        MIXPANEL_TOKEN: "${{ secrets. MIXPANEL_PROJECT_TOKEN }}"
```

Comment:
Is this a whitespace?

Reply:
Oops, slight issue with the replace tool. Thanks for catching that. Fixed